tests/test_config_parsing/test_res_config.py

```diff
@@ -32,17 +32,16 @@ def test_res_config_simple_config_parsing(tmpdir):
 LICENSE_PATH license
 """
         )
-    assert (
-        ResConfig(str(tmpdir + "/test.ert")).site_config
-        == ResConfig(
-            config_dict={
-                "CONFIG_DIRECTORY": str(tmpdir),
-                "DATA_FILE": "datafile",
-                "LICENSE_PATH": "license",
-                "RES_CONFIG_FILE": "test.ert",
-                "RUNPATH_FILE": "rpfile",
-            }
-        ).site_config
+
+    monkeypatch.chdir(tmpdir)
+    assert ResConfig("test.ert") == ResConfig(
+        config_dict={
+            "NUM_REALIZATIONS": 1,
+            "DATA_FILE": "datafile",
+            "LICENSE_PATH": "license",
+            "RES_CONFIG_FILE": "test.ert",
+            "RUNPATH_FILE": "/rpfile",
```

Comment:
is the `/` now needed here since it is added ? And why is it not needed on `licence` and `datafile`?

Reply:
No, not needed. Will remove